src/content/en/fundamentals/native-hardware/fullscreen/index.md

```diff
@@ -0,0 +1,481 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Going fullscreen.
+
+{# wf_updated_on: 2016-10-01 #}
+{# wf_published_on: 2016-10-01 #}
+
+# Making Fullscreen experiences {: .page-title }
+
+We have the ability to easily make immersive fullscreen web sites and 
+applications, but like anything on the web there are a couple of ways to do it.
+This is especially important now that more browsers are supporting an "installed
+web app" experience which launch fullscreen.
+
+<figure>
+<iframe width="324" height="576" src="//www.youtube.com/embed/ZRqr5x73-ng" frameborder="0" allowfullscreen></iframe>
+<figcaption>Example of a site going fullscreen in Chrome for Android</figcaption>
+</figure>
+
+## Getting your app or site fullscreen
+
+There are several ways that a user or developer can get an web app fullscreen.
+
+* Fake it: auto-hide the address bar
+* Request the browser to go fullscreen in response to a user gesture.
+* Install the app to the home screen
+
+### Fake it: auto-hide the address bar
+
+You can "fake fullscreen" by auto-hiding the address bar as follows:
+
+    window.scrollTo(0,1);
+
+Note: I am telling you this as a friend. It exists, it is a thing, but it 
+      is a hack. Please don't use it. &mdash; Paul
```

Comment:
I wanted to get this out of the way, because a lot of people do it even though it is not the preferred solution.